install_config/install/prerequisites.adoc

```diff
@@ -456,6 +456,21 @@ then the NetworkManager dispatch script does not create the relevant
 dnsmasq manually.
 //end::networkmanager[]
 
+[[install-config-network-using-firewalld]]
+==== Configuring firewalld as the firewall
+
+While iptables is the default firewall, firewalld is recommended for new
+installations. You can enable firewalld by setting `os_firewall_use_firewalld=true` in xref:../../install_config/install/advanced_install.adoc#advanced-install-configuring-firewalls[the Ansible inventory file].
+
+Setting this opens the required ports and adds rules to the default zone, which
+ensure that firewalld is configured correctly.
+
+[NOTE]
+====
+The firewalld default configuration cannot be overridden. This means that more
```

Comment:
I'm not sure what we are trying to say here... do you mean that you can't get ansible to set rules for a different zone?

You can use multiple zones as long as the interface that the nodes use to talk to one another is in the default zone.  i.e. if you had a storage network then you could put those interfaces into a different zone.

Reply:
Something like this, then? "Because you cannot override the firewalld default configuration, your advanced options are limited. For example, you cannot configure multiple zones. If you need to use multiple zones, you must place the network interface in the default zone."